README.md

```diff
@@ -7,3 +7,5 @@ Outputted files:
 - <https://alex.github.io/nyt-2020-election-scraper/battleground-state-changes.csv>
 
 Inspired by Simon Willison: https://simonwillison.net/2020/Oct/9/git-scraping/
+
+**IMPORTANT**: Contributions are welcome, but make sure you check the pull request template: https://github.com/fractionalhare/nyt-2020-election-scraper/blob/master/.github/pull_request_template.md
```

Comment:
```suggestion
**IMPORTANT**: Contributions are welcome, but make sure you check [the pull request template](.github/pull_request_template.md)
```
Perhaps use a relative link here? As described at https://github.blog/2013-01-31-relative-links-in-markup-files/

Also, my GitHub-fu is a bit rusty, but will this Markdown file auto-populate pull request descriptions once this is merged, or would contributors need to manually check the template?

Reply:
It will auto-populate in all new PRs.